src/utils.php

```diff
@@ -189,3 +171,8 @@ function getVendorDir(\stdClass $composerJson = null): string
 {
     return $composerJson->config->{'vendor-dir'} ?? 'vendor';
 }
+
+function strStartsWith(string $haystack, string $prefix): bool
+{
+    return empty($prefix) || strpos($haystack, $prefix) === 0;
```

Comment:
I find it easier to read with a helper function, especially since prefix is duplicated. Using substr would be a better implementation though.